test/python/primitives/test_backend_estimator.py

```diff
@@ -32,8 +31,6 @@
 
 
 BACKENDS = [
```

Comment:
Since this is just a singleton now, no need to use `@combine` anymore. Would be nice to if it's cleaned up.